crates/schemalate/src/firebolt/firebolt_schema_builder.rs

```diff
@@ -0,0 +1,255 @@
+use super::errors::*;
+use super::firebolt_types::{BasicType, Column, FireboltType, Table};
+
+use doc::inference::{ArrayShape, Shape};
+use doc::Annotation;
+use json::schema::{self, index::IndexBuilder, types};
+use serde_json::Value;
+
+pub const FAKE_BUNDLE_URL: &str = "https://fake-bundle-schema.estuary.io";
```

Comment:
I copied this from materialize-elasticsearch. Is there some better URL we can use?

Reply:
I remember this conversations before on the `schema uri`. 
https://estuaryworkspace.slack.com/archives/C012S5U0WP2/p1636758773029600?thread_ts=1636743849.020600&cid=C012S5U0WP2

Guess it is fine for now. And maybe we can switch to the `canonical url` in the future, as phil mentioned in the thread..